src/components/Inputs/Select.tsx

```diff
@@ -427,10 +459,30 @@ const SelectedValue = styled.div<InputProps>`
     `}
 `
 
+const CheckMark = styled.div`
+  height: 16px;
+  width: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: ${({ theme }) => theme.global.accent};
+  color: var(--color-white);
+  border-radius: 40px;
+  padding: 3px;
+`
+
```

Comment:
Let's move this to its own component so that we don't repeat the styled-component definition as long as the 3 line JSX?

Reply:
Fixed in #614